### PR TITLE
Add little note

### PR DESCRIPTION
**Pull request by Momoko-Sanchez**

### CODE REVIEW

File: _posts/0000-01-05-Momoko-Sanchez.md [hunk 1]
```diff
@@ -7,3 +7,5 @@ title: "Welcome to our slide deck!"
 {: .center}
 
 Caption here
+
+might want to look into that css later....
```
Comment (beardofedu) [inline _posts/0000-01-05-Momoko-Sanchez.md]: remove the `Caption here` and replace it with your `might want to look` content 👍

Comment (manas-mukherjee) [inline _posts/0000-01-05-Momoko-Sanchez.md]: which css ?

Comment (manas-mukherjee) [reply _posts/0000-01-05-Momoko-Sanchez.md]: Can you please mention the css filename ?

**Review by manas-mukherjee — DISMISSED** [review]
Overall the changes look good. If you add the filename, it will be perfect.

**Review by spoondla1 — COMMENTED** [review]
good. 👍

**Review by sridhar353 — APPROVED** [review]
approved

**Pull request merged**